controllers/toolchaincluster_resources/testdata/membersa.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: toolchaincluster-member
+  namespace: {{.Namespace}}
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-member
+  namespace: {{.Namespace}}
+rules:
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - "*"
+  verbs:
+  - "*"
```

Comment:
There are already some templates in https://github.com/codeready-toolchain/toolchain-common/tree/master/pkg/template/testdata so couldn't we use them?

Reply:
I've updated the tests to reuse the same templates in https://github.com/codeready-toolchain/toolchain-common/pull/373/commits/a6e5f881e6cf1fb6adba401619de36bf701aed82

Reply:
I haven't done this initially because I exposing the fs variables would make Go to include those in the final binary ( unless there's some other way to do it) . But I think it's not a big issue.

Reply:
I've moved those back to the test directory in https://github.com/codeready-toolchain/toolchain-common/pull/373/commits/8ee46aacee2e9c2a0bb3a24f7c0027556cc34dfc , as discussed , so that the templates are not bundled with the production binary. 

but maybe there's some other way to avoid duplicating those 🤔